micro-ui/web/micro-ui-internals/packages/libraries/CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Changelog
 
+## [1.8.18]  [21-Jul-2025]
+- Checking with ui-components as dependency 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)